File: src/ve/org/bcv/services/ParametroNominaService.java
package ve.org.bcv.services;

import java.util.List;

import javax.ejb.Local;

import ve.org.bcv.dto.BenefMedicamentoDto;
import ve.org.bcv.dto.ChartDto;
import ve.org.bcv.dto.ParametroNominaDto;
import ve.org.bcv.dto.ResultBoleanDto;
import ve.org.bcv.dto.UserEncuestaDto;
@Local
public interface ParametroNominaService {
	List<ParametroNominaDto> findByCoProcesoNom(String co_proceso_nom);
	boolean save(BenefMedicamentoDto benefMedicamentoDto,List<ParametroNominaDto> lista);
	UserEncuestaDto findUserEncuesta(String cedula);
	ResultBoleanDto personaYaEncuestada( String cedula,String nuCedulaFamiliar);
	ChartDto graficar(String interesdos);
	List<ParametroNominaDto> findByCoProcesoNomNo(String co_proceso_nom,String tx_parametro_nom) ;
	
}

File: WebContent/js/service/parametroNominaService.js
'use strict';

app.service('parametroNominaService',
    function parametroNominaService($resource) {
        return $resource('services/encuestaCtrl/:co_proceso_nom', {
        	co_proceso_nom: '@co_proceso_nom'
        }, {
            query: {
                method: 'GET',
                isArray: false
            },
            queryNo: {
            	url:'services/encuestaCtrl/listNo/:co_proceso_nom/:tx_parametro_nom',
                method: 'GET',
                isArray: false
            },
            findUser: {
            	url:'services/encuestaCtrl/findUser/:cedula',
                method: 'GET',
                isArray: false
            },
            personaYaEncuestada : {
            	url:'services/encuestaCtrl/personaYaEncuestada/:cedula/:nuCedulaFamiliar',
                method: 'GET',
                isArray: false
            }, 
            graficar: {
            	url:'services/encuestaCtrl/graficar/:interesdos',
                method: 'GET',
                isArray: false
            }  
        });
    });


 